src/app/components/options-sidebar/project-management/project-management.component.html

```diff
@@ -1,17 +1,7 @@
 <div class="parent">
-  
-  <app-create-project class="item"
-    [projectToEdit] = "project"
-    (savedProject)="updateProject($event)"
-    (cancelForm) = "cancelForm()"
-  >
-  </app-create-project>
-
-  <app-project-list class="item"
-  [projects] = "projects"
-  (editProject) = "editProject($event)"
-  (deleteProject) = "deleteProject($event)"
-  >
-  </app-project-list>
-</div>
+    <app-create-project class="item" [projectToEdit]="project" (savedProject)="updateProject($event)" (cancelForm)="cancelForm()">
+    </app-create-project>
```

Comment:
please, fix the indentation, remember that we are using an indentation based in 2 spaces.

Reply:
done